mod/metrics/lib/javascript/script_value_type.js.coffee

```diff
@@ -11,7 +11,7 @@ showAndHide = (slot, value) ->
       when 'Number'
         'number_details'
       when 'Money'
-        'currency_details'
+        'number_details'
```

Comment:
I can't approve this PR with this change in it, because it will break the value type functionality.

ALSO, this may be what is causing cukes to fail; you have to update the assets if you change js!